### ISSUE TITLE
frontend style

### ISSUE THREAD

**Issue by ThePianoDentist** [charlesastaylor/openPUBG#6]
a) what css framework do we want?
I think materialize.css is pretty nice http://materializecss.com/

b) do we want to use a fancy javascript framework, i.e. react.js. its probably overkill, main advantages you would get would be not needing to reload page when stats/leaderboards update. But are updates going to be frequent enough to matter?
Other advantage is I believe it makes converting into an app almost no extra effort
It's another thing to learn which is both a positive and a negative.

*react.js handles the `view` part, it doesnt handle passing the data around, so you need something like flux, alt.js or redux.js to do that part....as I said its almost certainly overkill

**Comment by charlesastaylor** [comment]
a) I'm down to use materialzililsilzis. Still not used a framework at all, not even bootstrap :)
b) do you need a fancy js framework to have updates without reloads? doesnt just jquery handle ajax?
and in terms of converting to app will we ever want to? imo if you just make your web app mobile friendly enough why do you need a dedicated app?

**Comment by ThePianoDentist** [comment]
Yeah jquery is fine. It's not cool anymore, but it does the job.

You can usually write things more succinctly and reuse js code more if not
using jquery. But should be small enough to not matter

**Comment by ThePianoDentist** [comment]
hmmm. if we are doing api style. very basic page - all data loaded in by api calls....might want to go back to react. I said jquery fine, because I thought we'd be inserting the data directly into the template with python/jinja

http://chibicode.com/react-js-introduction-for-people-who-know-just-enough-jquery-to-get-by/
https://realpython.com/blog/python/the-ultimate-flask-front-end/

If we're planning to actually do it properly..and this be a long term project, I think we should use react. once we're used to it, it's harder to make bugs than jquery(the way jquery handles events means can wind up with 'unexpected' behaviour way more often) and easier to make updates/modifications

**Comment by ThePianoDentist** [comment]
tl; dr FUCK IT. well use jquery

I have flip-flopped again.

All we are doing with data is getting it from backend.
except for making an account to view personalised data. their is no user-input.

therefore there are no confusing sets of event-handlers from user input -> backend -> user view
that end up spaghetti-fied with jquery.

there is actually no solid reasoning to prefer a react/flux architecture over jquery here.

im committing my start of using react on a separate branch
